dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,10 +1,43 @@
 class RubricsController < ApplicationController
+  include Rails.application.routes.url_helpers
+
   def edit
     @rubric = Rubric.find_by(id: params[:id])
     @lesson = @rubric.lesson
   end
 
   def new
     @lesson = Lesson.find_by(id: params[:lessonId])
+    @rubric = Rubric.new
+  end
+
+  def create
+    @rubric = Rubric.new(rubric_params)
+    @lesson = @rubric.lesson
+    if @rubric.save
+      redirect_to(edit_rubric_path(@rubric.id))
+      return @rubric.id
```

Comment:
This behavior still confuses me. IMO, I think it would be more readable if you did something like
```
render json: {redirectUrl: edit_rubric_path(@rubric.id)}
```
then updated the javascript to match. Thoughts?